addon-test-support/@ember/test-helpers/dom/-is-select-element.ts

```diff
@@ -0,0 +1,14 @@
+import { isDocument } from './-target';
+
+/**
+  @private
+  @param {Element} element the element to check
+  @returns {boolean} `true` when the element is a select element, `false` otherwise
+*/
+export default function isSelectElement(element: Element | Document): element is HTMLSelectElement {
+  return (
+    !isDocument(element) &&
+    element.tagName === 'SELECT' &&
+    (element as HTMLSelectElement).type !== 'hidden'
```

Comment:
Hey sorry for the late reply, I added this because I wanted to keep it as close to how the fillIn in currently works and i noticed on the form-control helper also used hidden so i thought i might be expected behaviour that interfacing with hidden elements is to be discouraged. But happy to remove